src/install.rst

```diff
@@ -83,21 +89,6 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
             .. group-tab:: Native
 
-               .. warning::
```

Comment:
This should still be linked somewhere

Reply:
The option is still available if you click the linked FAQ page which says:

https://github.com/nextstrain/docs.nextstrain.org/blob/5b5f71fcfa65463c3920cfbdd340814d2be9e5d9/src/reference/faq.rst?plain=1#L23-L29

Though it's now simplified to avoid an intermediate `base_osx-64` environment – something I got from https://github.com/conda-forge/miniforge/issues/165#issuecomment-860233092 but only caused more confusion in practice.

Reply:
I see you created #125 for this. Let's continue discussion there.